var/spack/repos/builtin/packages/wrf/package.py

```diff
@@ -157,7 +163,11 @@ class Wrf(Package):
     patch("patches/4.2/derf_fix.patch", when="@4.2 %aocc")
 
     patch("patches/4.4/arch.postamble.patch", when="@4.4:")
-    patch("patches/4.4/configure.patch", when="@4.4:")
+    patch("patches/4.4/configure.patch", when="@4.4:4.4.2")
+
+    patch("patches/4.5/configure.patch", when="@4.5:")
+    patch("patches/4.5/adios2-remove-deprecated-functions.patch", when="@4.5: ^adios2@2.9:")
```

Comment:
Can you add brief comments explaining what is the effect of the patch, why it is needed and where it was taken from?

Reply:
Done.